client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ContactDetails.tsx

```diff
@@ -16,13 +16,10 @@ const ContactDetails = (props: Props) => {
     const { errors } = useFormContext();
     const { index , interactedContact } = props;
 
-    const formErrors = errors.form ? errors.form[index] : {};
-    let formHasErrors = false;
-    for(let key in formErrors) {
-        if(formErrors[key] !== undefined) {
-            formHasErrors = true;
-        }
-    }
+    const formErrors = errors.form ? (errors.form[index] ? errors.form[index] : {}) : {};
+    const formHasErrors = Object.entries(formErrors)
+        .map(([key, value]) => value)
+        .some((value) => value !== undefined);
```

Comment:
you dont need to loop twice, you could do the some check w/o the map 🤔 
`const formHasErrors = Object.entries(formErrors).some(([key, value]) => value !== undefined);`

Reply:
whoops , fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/998/commits/9e0e295ec2466dcaed92c80e2143ecde9e63d9dc)